include/fem-dem/vans_assemblers.h

```diff
@@ -690,6 +690,48 @@ class VANSAssemblerFPI : public NavierStokesAssemblerBase<dim>
   const Parameters::CFDDEM cfd_dem;
 };
 
+/**
+ * @brief Class that assembles the fluid_particle interactions (FPI) for the
+ * VANS equations such as the drag force, when the forces are projected from the
+ * particles to the fluid grid.
+ *
+ * @tparam dim An integer that denotes the number of spatial dimensions
+ *
+ * @ingroup assemblers
+ */
+
+template <int dim>
+class VANSAssemblerFPIProj : public NavierStokesAssemblerBase<dim>
```

Comment:
```suggestion
class VANSAssemblerFPIProjection : public NavierStokesAssemblerBase<dim>
```

I think it's better to avoid abbreviations in class name. We can be verbose especially when the meaning of the word is important.